### PR TITLE
docs: adding azure to supported OS grid

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

n/a

## Description of changes being made
This came up in Konv standup - 
As we support the default configuration for Azure builds, this should be added to this grid.

FYI @JohnB64-JBV 

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by D2iQdocs — APPROVED** [review]

**Comment by msdolbey** [comment]
Turns out - that outisde of the pipes just didn't work at all, but, it was getting caught up by a different linter that just ... wasn't showing up with the pre-commit (there was vale.spelling AND vale.terms, so, added vale.terms).

File: pages/dkp/konvoy/2.1/supported-operating-systems/index.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ Konvoy supports the following base Operating Systems.
 ## Amazon Web Services (AWS)
 
 <!-- vale Vale.Spelling = NO -->
-| Operating System      | Kernel                      | Default Config | FIPS | Air Gapped | FIPS with Air Gapped | GPU Support <!-- vale Vale.Spelling = YES --> |
+<!-- vale Vale.Terms = NO -->
+| Operating System      | Kernel                      | Default Config | FIPS | Air Gapped | FIPS with Air Gapped | GPU Support <!-- vale Vale.Spelling = YES --> <!-- vale Vale.Terms = YES --> |
```
Comment (vacas5) [inline pages/dkp/konvoy/2.1/supported-operating-systems/index.md:19]: this seems highly unusual. You said the comments won't work below the table?

**Comment by cmastr** [comment]
This is on the wrong branch @msdolbey

File: pages/dkp/konvoy/2.1/supported-operating-systems/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.1/supported-operating-systems/index.md:19]: I just wanted it to take affect for this one line is all.

Comment (cmastr) [reply pages/dkp/konvoy/2.1/supported-operating-systems/index.md:19]: I think to keep it very limited in scope, in the table is good.

Comment (msdolbey) [reply pages/dkp/konvoy/2.1/supported-operating-systems/index.md:19]: So, yeah - @vacas5 - the vale linter is happy again now.

**Review by cmastr — APPROVED** [review]

File: pages/dkp/konvoy/2.1/supported-operating-systems/index.md (hunk 1 above)
Comment (vacas5) [reply pages/dkp/konvoy/2.1/supported-operating-systems/index.md:19]: I just want to make sure it doesn't render in the preview

**Review by vacas5 — APPROVED** [review]

**Pull request merged**